scripts/install2

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
+# s_h_e_l_lcheck disable=SC1091
```

Comment:
why the `_`?

Reply:
to disable the disable

Line removed as its not needed